contracts/lib.rs

```diff
@@ -1213,6 +1218,15 @@ mod prosopo {
             }
             provider_ids
         }
+
+        fn get_random_number(&self, min: u64, max: u64) -> u64 {
+            let random_seed = self.env().random(&b"prosopo_contract"[..]);
```

Comment:
The seed might want to optionally be the calling account (`self.env().caller();`) for additional randomness. Otherwise use the hard coded value `prosopo contract`. Great work getting this to work as a seed. The conversion is complicated.

Reply:
pushed!